src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    operate(operation, number) {
+      operation.call(this, number);
+
+      return this;
+    },
```

Comment:
Nice solution
also, you can do like that

```suggestion
    operate(callback, number) {
      this.result = callback(number)

      return this;
    },
```